make/ci.mk

```diff
@@ -1,5 +1,7 @@
+__PYTHON := python3
```

Comment:
question: what is the rationale?

Reply:
Pretty much just to make it easier to point a different python down the road